client/my-sites/current-site/index.jsx

```diff
@@ -66,37 +72,54 @@ class CurrentSite extends Component {
 
 		return (
 			<Card className="current-site">
-				{ this.props.siteCount > 1 && (
-					<span className="current-site__switch-sites">
-						<Button borderless onClick={ this.switchSites }>
-							<Gridicon icon="chevron-left" />
-							<span className="current-site__switch-sites-label">
-								{ translate( 'Switch Site' ) }
-							</span>
-						</Button>
-					</span>
-				) }
+				<div
+					role="button"
+					tabIndex={ 0 }
+					aria-hidden="true"
```

Comment:
Can you explain if this is accessible and if so why? Often messing with tabindex directly is a cause of UX and a11y bugs.

Reply:
There is no need to be accessible, it is just an overlay above the 3 buttons area
![](https://cln.sh/dl6oyW+)
Tabbing through these 3 buttons (which have semantics) and clicking enter works.